src/include/setup_particles.hxx

```diff
@@ -46,6 +46,7 @@ struct SetupParticles
       }
       return n_prts;
     }
+    //member function returns int, cannot return this
     return npt.n / norm_.cori + .5;
```

Comment:
I'm not quite sure what you mean by this comment, as the value can (does) return the expression. You can change this to an explicit `return int(...)` if you got a warning or otherwise prefer to be explicit about the conversion to int that happens when returning the value.

Reply:
I guess I just don't understand the logic,  if we aren't returning fractional why would we add .5 just to have the result cast to int?

Reply:
To make it round things properly. (2.2 particles -> 2 particles, but 2.6 particles -> 3 particles)